addons/project/models/project.py

```diff
@@ -918,7 +918,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
         help="Sum of the time planned of all the sub-tasks linked to this task. Usually less than or equal to the initially planned time of this task.")
     # Tracking of this field is done in the write function
     user_ids = fields.Many2many('res.users', relation='project_task_user_rel', column1='task_id', column2='user_id',
-        string='Assignees', default=lambda self: self.env.user)
+        string='Assignees', default=lambda self: self.env.user, context={'active_test': False})
```

Comment:
Hello @IT-Ideas 😄 
I wonder if it would be possible to make archived users display if they are already set but not able to select an archived user from the field?
Maybe by setting a domain on the view's declaration aswell? `('active', '=', True)`
I don't know if we could previously select archived users but I guess it has never been and I suspect there are a lot of archived users we should not see on prod 😄

Reply:
Indeed adding the domain on the view could help filtering them.